upstairs/src/lib.rs

```diff
@@ -5584,32 +5574,6 @@ impl Upstairs {
         }
     }
 
-    /*
-     * Look to see if this specific downstairs is in LiveRepair, and if so,
-     * apply special consideration for dependencies, as they could be unique
-     * for this downstairs.
-     *
-     * First off, any job that was "skipped" should not be a dependency for
-     * this specific downstairs.  In addition, any job that happened before
-     * the skipped jobs that was marked as "Done" should also be removed, as
```

Comment:
We should harvest the comment here starting from "In addition, .." and add it, I think,
to the comment above `remove_dep_if_live_repair()`.

It's another subtle point about Live Repair that I don't want to forget.

Reply:
Good catch, I added this comment to the `remove_dep_if_live_repair` docstring